osm2lanes-web/src/control.rs

```diff
@@ -52,11 +52,11 @@ impl Component for Control {
                 let examples: BTreeMap<_, _> = tests
                     .into_iter()
                     .filter_map(|t| {
-                        let example_name = t.example().map(|e| e.to_owned());
+                        let example_name = t.example().map(std::borrow::ToOwned::to_owned);
```

Comment:
Less readable IMO, but I'm not here to argue with clippy!

Reply:
yeah, it's strange that `.to_owned` is in scope by default, but `ToOwned::to_owned` is not.